vendor/vendor.json

```diff
@@ -617,6 +737,18 @@
 			"revision": "4e0d7dc8888fbb59764060e99b7b68e77a6f9698",
 			"revisionTime": "2018-10-14T14:49:52Z"
 		},
+		{
+			"checksumSHA1": "0cytTxS0qPVjtknc7OrfuNmxvBE=",
+			"path": "github.com/dgrijalva/jwt-go",
```

Comment:
Just want to double check, are these libraries all used for azure module?

Reply:
yes, dependencies were added for the azure sdk